tests/test_api.py

```diff
@@ -1521,36 +1521,6 @@ def test_not_in_relation_float(self):
         result = self._id_in_result('Version', filters, self.version['id'])
         self.assertFalse(result)
 
-    def test_in_relation_comma_list(self):
-        """
-        Test that 'in' relation using commas (old format) works with list fields.
-        """
-        filters = [['sg_priority', 'in', self.ticket['sg_priority'], '1'],
-                   ['project', 'is', self.project]]
-
-        result = self._id_in_result('Ticket', filters, self.ticket['id'])
```

Comment:
Not equivalent with Version on these ones ?

Reply:
Good call! I just replace them and got these tests back. Thank you!